test/new_relic/agent_test.rb

```diff
@@ -175,8 +175,7 @@ def test_instance
 
     def test_register_report_channel
       NewRelic::Agent.register_report_channel(:channel_id)
-      assert NewRelic::Agent::PipeChannelManager.channels[:channel_id] \
-        .kind_of?(NewRelic::Agent::PipeChannelManager::Pipe)
+      assert_kind_of(NewRelic::Agent::PipeChannelManager::Pipe, NewRelic::Agent::PipeChannelManager.channels[:channel_id])
```

Comment:
For all new code, please keep lines at or below 120 columns in length.

Reply:
Caught again! The whole file is now [sparkling clean](https://github.com/newrelic/newrelic-ruby-agent/pull/1484/commits/15172e1f11bb6b361397be96ff82e15a51f75c0f) without any 120 violations.